cryptonite.cabal

```diff
@@ -298,28 +299,40 @@ Library
   else
     include-dirs:      cbits/include32
 
-  if arch(x86_64) || arch(aarch64)
-    C-sources:         cbits/decaf/p448/arch_ref64/f_impl.c
+  if arch(x86_64)
```

Comment:
This nested `if` is required since the Cabal version floor for this package is currently 1.8.